pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -86,11 +86,17 @@ social:
               %h4
                 =hoc_s(:front_join_us_heading)
               =hoc_s(:front_join_us_description_short)
-            #joinustext{style: "font-size: 18px; font-weight: 500; padding-top:20px;"}
-              %div{style: "width: 100%; text-align: center"}
-                %a#joinbutton{onclick: "adjustScroll('hocsignupform')", target: "_blank", style: "text-decoration:none"}
-                  %button{style: "font-size: 18px; font-weight: 500; padding-left: 30px; padding-right: 30px; height: 40px;"}
-                    =hoc_s(:front_join_us_button)
+            %div{style: "text-align: center;"}
+              %div#signuptext{style: "display: inline-block; font-size: 18px; font-weight: 500; padding:20px 4px 0px 4px;"}
```

Comment:
Can you take out the inline css and put into a new file in hoc.com/public/css? We still have a lot of files with inline css and we're trying to remove that any time we update a file.

Then they can share classes like `#ctatext` and `#ctabutton` (call to action) since they're exactly the same.